core/driver/sources/src/main/scala/org/virtuslab/ideprobe/reporting/ErrorValidator.scala

```diff
@@ -3,13 +3,20 @@ package org.virtuslab.ideprobe.reporting
 import org.virtuslab.ideprobe.config.CheckConfig
 import org.virtuslab.ideprobe.protocol.IdeMessage
 
+import scala.language.postfixOps
+
 object ErrorValidator {
   def apply(config: CheckConfig, errors: Seq[IdeMessage]): Option[Exception] = {
-    if (errors.isEmpty) None
+    val filteredErrors = errors
+      .filterNot(error => config.errors.includeMessages.exists( _.r findFirstIn error.content isEmpty))
```

Comment:
Hmmm I'd rather stick to `nonEmpty` as `isEmpty` is kinda-negation, which makes analyzing the expression harder